roles/ceilometer-common/tasks/logging.yml

```diff
@@ -9,3 +9,6 @@
       - compress
       - notifempty
       - delaycompress
+
+- name: remove service default log rotate for ceilometer
```

Comment:
are these logrotate files faulty ?   what is the reason we can't use them as is ?